### DISCUSSION TITLE
[Linux] Cannot build poetry without depending on poetry

### DISCUSSION THREAD

**Discussion by ghost** [python-poetry/poetry#5677] [General] [answered]
I wanted to start building a releasable `.deb` or `AppImage` of poetry, but I cannot seem to find a (documented) way of building poetry without installing poetry first.

./Makefile pull a docker image with multiple python versions and runs [./make-linux-release.sh](https://github.com/python-poetry/poetry/blob/master/make-linux-release.sh) but that installs poetry itself in order to package poetry.

How can poetry be built without depending on poetry?

**Comment by abn** [comment, marked as answer]
The recommended installation for Poetry today is either using https://install.python-poetry.org or using `pipx install poetry`. You can likely follow the instructions at https://python-appimage.readthedocs.io/en/latest/ to build an app image with Poetry installed.

For debs you can see https://packages.debian.org/sid/python3-poetry.
